orderbook/src/fee.rs

```diff
@@ -88,18 +88,25 @@ impl MinFeeCalculator {
 
     async fn compute_min_fee(&self, token: H160) -> Result<Option<U256>> {
         let gas_price = self.gas_estimator.estimate().await?;
+        let fee_in_eth = gas_price * GAS_PER_ORDER;
```

Comment:
Assuming that `GAS_PER_ORDER` is a constant upper bound on the maximum gas an order could cost, how did we arrive at this value? It could also be that the constant is really the gas cost for just settling the single order, but wouldn't it be more appropriate to factor in the distributed fee incurred by executing a bunch of different interactions. If so, then we would also have to limit the scope of interactions, since they can essentially run arbitrary code. With this in mind, I would think that the gas per order is some sort of upper bound determined by the block gas limit over the max number of orders which can be matched.